tests/voq/test_voq_nbr.py

```diff
@@ -220,21 +224,24 @@ def disable_nbr_bgp_neighs(node=None, results=None):
             'disable neighbors {} on neighbor host {}'.format(node['conf']['bgp']['peers'], node['host'].hostname))
         for peer in node['conf']['bgp']['peers']:
             for neighbor in node['conf']['bgp']['peers'][peer]:
-                node_results.append(node['host'].eos_config(
-                    lines=["neighbor %s shutdown" % neighbor],
-                    parents=['router bgp {}'.format(node['conf']['bgp']['asn'])],
-                    module_ignore_errors=True)
-                )
-                if ":" in neighbor:
+                if isinstance(node['host'], EosHost):
                     node_results.append(node['host'].eos_config(
-                        lines=["ipv6 route ::/0 %s " % neighbor],
+                        lines=["neighbor %s shutdown" % neighbor],
+                        parents=['router bgp {}'.format(node['conf']['bgp']['asn'])],
                         module_ignore_errors=True)
                     )
+                    if ":" in neighbor:
+                        node_results.append(node['host'].eos_config(
+                            lines=["ipv6 route ::/0 %s " % neighbor],
+                            module_ignore_errors=True)
+                        )
+                    else:
+                        node_results.append(node['host'].eos_config(
+                            lines=["ip route 0.0.0.0/0 %s " % neighbor],
+                            module_ignore_errors=True)
+                        )
                 else:
-                    node_results.append(node['host'].eos_config(
-                        lines=["ip route 0.0.0.0/0 %s " % neighbor],
-                        module_ignore_errors=True)
-                    )
+                    node_results.append(node['host'].shell("sudo vtysh -c 'configure terminal' -c 'router bgp " + str(node['conf']['bgp']['asn'])+ "' -c 'neighbor {} shutdown'".format(neighbor)))
```

Comment:
we don't need to add default route if its sonic device?

Reply:
Removed it from eos.